lib/mongodb.js

```diff
@@ -448,6 +458,16 @@ MongoDB.prototype.toDatabase = function(modelName, data) {
         coordinates: [data[p].lng, data[p].lat],
         type: 'Point',
       };
+    // SPIKE: Ideally coercePropertyValue should handle this but it does not have enough info
+    } else if (prop && prop.type === String) {
```

Comment:
Please try find a way how to use the existing code for converting `string` to `Decimal128`.